src/views/HomeView.vue

```diff
@@ -31,147 +31,25 @@
-          <li><code class="bg-base-300 ml-4">~/.config/StardewValley/Saves</code></li>
-          <li>4. Select a save folder, and choose the file that takes the following format:</li>
-          <li><code class="bg-base-300 ml-4">{player name}_{lots of numbers}</code></li>
-          <li>5. Submit the file down below.</li>
-        </ul>
-      </div>
-      <div v-else-if="activeTab === 'windows'" class="bg-base-200 rounded-lg">
-        <ul class="space-y-3 p-3">
-          <li class="flex items-center">
-            1. Press
-            <kbd class="kbd mx-2"><font-awesome-icon icon="fa-brands fa-windows" /></kbd> + <kbd class="kbd mx-2">r</kbd> to open "Run"
-          </li>
-          <li>2. In the resulting text field, enter the following file path:</li>
-          <li><code class="bg-base-300 ml-4">%appdata%\StardewValley\Saves</code></li>
-          <li>3. Select a save folder, and choose the file that takes the following format:</li>
-          <li><code class="bg-base-300 ml-4">{player name}_{lots of numbers}</code></li>
-          <li>4. Submit the file down below.</li>
-        </ul>
-      </div>
-      <div class="mt-3"></div>
-      <a class="text-sm italic hover:brightness-75" href="https://stardewvalleywiki.com/Saves" target="_blank">
-        Click here if you are unable to locate your save file based on the provided instructions.
-      </a>
-      <div class="modal-action flex justify-between">
-        <input
-          type="file"
-          :class="['file-input w-full max-w-xs bg-base-200 hover:brightness-90', fileError ? 'input-error' : '']"
-          @change="handleFileUpload"
-        />
-        <label for="my-modal" :class="['btn', isFileLoading ? 'loading' : '']" @click="submitSaveFile">Import</label>
-      </div>
-    </div>
+    <ImportSaveModal ref="importSaveModal" />
   </div>
 </template>
 <script setup lang="ts">
 import DateTracker from "@/components/DateTracker.vue";
 import VillagerGrid from "@/components/VillagerGrid.vue";
 import TrackingList from "@/components/TrackingList/TrackingList.vue";
 import InventoryGrid from "@/components/InventoryGrid.vue";
+import ImportSaveModal from "@/components/ImportSaveModal.vue";
 import store from "@/store";
 import { Ref, ref } from "vue";
-import { XMLParser } from "fast-xml-parser";
-
-const activeTab = ref("apple");
-const modalOpen = ref(false);
-const isFileLoading = ref(false);
-const fileError = ref(false);
-const saveFile: Ref<File | null> = ref(null);
 
-function tabPressed(os: string) {
-  switch (os) {
-    case "apple":
-      activeTab.value = "apple";
-      break;
-    case "windows":
-      activeTab.value = "windows";
-      break;
-  }
-}
-
-function handleFileUpload(event: Event) {
-  const inputElement = event.target as HTMLInputElement;
-  if (inputElement.files) {
-    const file = inputElement.files[0];
-    saveFile.value = file;
-    fileError.value = false;
-  }
-}
-
-function submitSaveFile() {
```

Comment:
this can be more robust with error messages but i don't think that's super necessary